vcf-dumper/vcf-dumper-lib/src/test/java/uk/ac/ebi/eva/vcfdump/VariantExporterTest.java

```diff
@@ -114,36 +125,94 @@ public static void tearDownClass() throws UnknownHostException {
         VariantExporterTestDB.cleanDBs();
     }
 
-    @Test
-    public void getSources() {
-        VariantExporter variantExporter = new VariantExporter();
 
+    @Test
+    public void getSourcesOneStudyNoFiles() {
         // one study
         String study7Id = "7";
         List<String> studies = Collections.singletonList(study7Id);
-        Map<String, VariantSource> sources = variantExporter.getSources(variantSourceDBAdaptor, studies);
+        List<VariantSource> sources =
+                variantExporter.getSources(variantSourceDBAdaptor, studies, Collections.EMPTY_LIST);
         assertEquals(1, sources.size());
-        VariantSource file = sources.get(study7Id);
+        VariantSource file = sources.get(0);
+
         assertEquals(study7Id, file.getStudyId());
         assertEquals("6", file.getFileId());
+    }
 
+    @Test
+    public void getSourcesTwoStudiesNoFiles() {
```

Comment:
With "no files", do you mean there is no file filter? In that case getSourcesTwoStudiesWithEmptyFilesFilter could improve consistency on naming across the test methods.